src/Module/Delegation.php

```diff
@@ -118,7 +118,7 @@ public static function content(array $parameters = [])
 			throw new ForbiddenException(DI::l10n()->t('Permission denied.'));
 		}
 
-		$identities = DI::app()->identities;
+		$identities = User::identities(DI::session()->get('submanage') ?: local_user());
```

Comment:
`Session->get()` already has a fallback: 
```suggestion
		$identities = User::identities(DI::session()->get('submanage', local_user()));
```